dorado/particle_track.py

```diff
@@ -814,16 +962,16 @@ def unstruct2grid(coordinates,
 
     gridX, gridY = np.meshgrid(xvect, yvect)
 
-    inputXY = scipy.array([x[:], y[:]]).transpose()
+    inputXY = np.array([x[:], y[:]]).transpose()
 
-    gridXY_array = scipy.array([scipy.concatenate(gridX),
-                                scipy.concatenate(gridY)]).transpose()
-    gridXY_array = scipy.ascontiguousarray(gridXY_array)
+    gridXY_array = np.array([np.concatenate(gridX),
+                             np.concatenate(gridY)]).transpose()
+    gridXY_array = np.ascontiguousarray(gridXY_array)
```

Comment:
Remind me to check and make sure this still works with numpy arrays. It probably will, but the code I based this on used scipy arrays, which I figured might interact with the interpolation function in some way.

Reply:
It should be the same, the `scipy` versions are just being retired and threw warning messages. The solution to get rid of the warnings was to switch to `numpy`.